enterprise/j2ee.common/test/unit/src/org/netbeans/modules/j2ee/common/ClasspathUtilTest.java

```diff
@@ -27,13 +27,15 @@
 import java.util.LinkedHashMap;
 import java.util.LinkedList;
 import java.util.List;
+import org.junit.Ignore;
 import org.netbeans.junit.NbTestCase;
 import org.openide.filesystems.FileUtil;
 
 /**
  *
  * @author sherold
  */
+@Ignore("jar files which are base for this class are missing")
 public class ClasspathUtilTest extends NbTestCase {
```

Comment:
does ignore work?

CI is testing `j2ee.core` but not `j2ee.common`:
https://github.com/apache/netbeans/blob/5af5f30e97c08222539b0c9ce79afc19e1a52b68/.github/workflows/main.yml#L1988-L1989

feel free to register it in the yaml.

Reply:
I'm not 100% sure about CI. When I run the tests locally via IDE it is ignored. I wonder if it's worth keeping this test anyway - it seems the test is broken since (at least) git migration.

Do you know anything about:
1) Why these jars were removed (license issue?)
2) Can they be recovered?

Reply:
I edited my post while you commented. please check again :)

> When I run the tests locally via IDE it is ignored. 

thats good